### DISCUSSION TITLE
Build Process - Behind the Scenes ?

### DISCUSSION THREAD

**Discussion by uneknown** [vercel/vercel#8479] [General] [answered]
I use Vercel on a daily basis. I love it! But there is something that's bugging me. There are no resources provided on how the build process works behind the scenes! For stuff that I use daily, I like to know what happens in the background 😄 What is Vercel using to provision the build environments? How are they isolated? What does the general architecture look like?

It would be great if someone could outline this to me!

**Comment by amyegan** [comment, marked as answer]
This post is a duplicate. Please continue the conversation at [github.com/orgs/vercel/discussions/804](https://github.com/orgs/vercel/discussions/804)
